Makefile

```diff
@@ -170,6 +171,10 @@ superuser: ## create a Django superuser
 	@$(MANAGE) createsuperuser
 .PHONY: superuser
 
+shell: ## connect to database shell
+	@$(MANAGE) shell_plus
```

Comment:
I do not know django-extensions. What are the benefits? And particularly in the case of `shell_plus` compare to the `shell` command offered by Django ?

Reply:
I usually use:
* `shell_plus`, it's like `shell` but it also import automatically all the models of your project when you open a shell
* `reset_db` remove all the contain of your database
* `list_signals` to have a list of all signal of the application
there are other commands that I do not use personally https://django-extensions.readthedocs.io/en/latest/command_extensions.html

Reply:
Interesting, let's give it a try